test/test_ea_mu_plus_lambda.py

```diff
@@ -57,3 +57,97 @@ def objective(ind):
 
     for idx, ind in enumerate(offsprings):
         assert ind.idx == len(pop.parents) + idx
+
+
+def test_n_offspring_less_than_n_breeding():
+    n_offsprings = 10
+    n_breedings = 5
+    with pytest.raises(ValueError):
+        _ = cgp.ea.MuPlusLambda(n_offsprings, n_breedings, 1)
+
+
+def test_fitness_wrong_return_type(population_params, genome_params):
+    def objective(individual):
+        individual.fitness = 5
+        return individual
+
+    pop = cgp.Population(**population_params, genome_params=genome_params)
+
+    ea = cgp.ea.MuPlusLambda(10, 10, 1)
```

Comment:
would it make sense to use the `ea_params` fixture here and in other places?

Reply:
Yes! Done.